ga4gh/server/repo/models.py

```diff
@@ -86,7 +79,6 @@ class Biosample(BaseModel):
     individualAgeAtCollection = pw.TextField(null=True)
 
     class Meta:
-        db_table = 'Biosample'
```

Comment:
hm, are these `db_table` attributes meaningful, even though they are not used by us? (i.e. used as a directive to some other library)